README.md

```diff
@@ -40,6 +40,13 @@ docker-compose up -d
 rails db:setup
 
 rails server
+
+# to set git hooks
```

Comment:
Could this be moved into it's own section under teh heading Git Hooks.

Reply:
Done: https://github.com/wearefuturegov/beacon/pull/101/commits/a9fa4989afbe0e9b4ff0639e8eed5ca77f5e4c77